core/src/apps/common/cbor.py

```diff
@@ -271,40 +274,10 @@ def encode_streamed(value: Value) -> Iterator[bytes]:
     return _cbor_encode(value)
 
 
-def encode_chunked(value: Value, max_chunk_size: int) -> Iterator[bytes]:
```

Comment:
was this unused?

Reply:
Yes, it was unused in the production code. As it was however used in tests, I copy-pasted it there